examples/component-example/src/main/java/io/dekorate/examples/component/Main.java

```diff
@@ -22,13 +22,10 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@ComponentApplication(name = "hello-spring-boot", exposeService = true, envs = @Env(name = "key1", value = "val1"))
-@Link(name = "should-be-overridden-by-properties", componentName = "should-be-overridden-by-properties")
-@SpringBootApplication
```

Comment:
I would keep spring, as its a nice way of show casing how the `runtime`, `version` properties of component are popullated.

The rest I agree that should go.

Reply:
Ok, I agree, I hadn't thought about it. I'm gonna keep spring :-)